integreat_cms/api/tests/expected-outputs/regions_live.json

```
@@ -0,0 +1,30 @@
+[
+    {
+        "id": 1,
+        "name": "City Augsburg",
```

```suggestion
        "name": "Stadt Augsburg",
```
The language of the prefix should always be in the default language of the region. If it is not, we have found a bug in the API.

---

Strange, only circle ci api returns here "city", my local dev api return "Stadt"
```
FAIL: test_api_regions (integreat_cms.api.tests.api_tests.ApiTest)
Test if the view ``api_regions`` returns the HTTP status code ``200`` with correct data
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/home/circleci/project/integreat_cms/api/tests/api_tests_utils.py", line 33, in test_function
    self.assertEqual(regions, response.json())
AssertionError: Lists differ: [{'id': 1, 'name': 'Stadt Augsburg', 'path': 'augsburg', 'liv[514 chars]"]'}] != [{'id': 1, 'name': 'City Augsburg', 'path': 'augsburg', 'live[512 chars]"]'}]

First differing element 0:
{'id': 1, 'name': 'Stadt Augsburg', 'path': 'augsburg', 'liv[224 chars]g"]'}
{'id': 1, 'name': 'City Augsburg', 'path': 'augsburg', 'live[222 chars]g"]'}
```

---

Both use the same fixture, so I think it is something setting/database related :thinking:

---

No, actually the translation file is not existing in this step at circle ci

---

Ahhhh ok this makes sense. Thanks for digging into that rabbit hole :rabbit: